UnityProject/Assets/Runtime/PhxEnvironment.cs

```diff
@@ -329,31 +328,34 @@ public void Tick(float deltaTime)
         for (int i = 0; i < Loading.Count; ++i)
         {
             ELoadStatus status = EnvCon.GetStatus(Loading[i].Handle);
+
             if (status == ELoadStatus.Loaded)
             {
                 LVL scheduled = Loading[i];
-                if (!scheduled.bIsSoundBank)
-                {
-                    var lvl = EnvCon.GetLevel(Loading[i].Handle);
-                    Debug.Assert(lvl != null);
-                    scheduled.Level = lvl;
+                
+                var lvl = EnvCon.GetLevel(Loading[i].Handle);
+                Debug.Assert(lvl != null);
+                scheduled.Level = lvl;
 
-                    if (lvl.IsWorldLevel)
+                if (lvl.IsWorldLevel)
+                {
+                    if (WorldLevel != null)
                     {
-                        if (WorldLevel != null)
-                        {
-                            Debug.LogErrorFormat("Encounterred another world lvl '{0}' in environment! Previously found world lvl: '{1}'", lvl.Name, WorldLevel.Name);
-                        }
-                        else
-                        {
-                            WorldLevel = lvl;
-                        }
+                        Debug.LogErrorFormat("Encounterred another world lvl '{0}' in environment! Previously found world lvl: '{1}'", lvl.Name, WorldLevel.Name);
+                    }
+                    else
+                    {
+                        WorldLevel = lvl;
                     }
-
-                    // grab lvl localizations, if any
-                    Localizations.AddRange(lvl.Get<Localization>());
                 }
 
+                // grab lvl localizations, if any
+                Localizations.AddRange(lvl.Get<Localization>());
+                
+
+                SoundLoader.Instance.InitializeSoundProperties(lvl);
```

Comment:
Must be done whenever a `Level` is loaded so that the sound loader can map ODF sound names -> `SoundProperty` `Field`s in `snd_` `Config`s -> samples referenced in the `snd_` `Field`'s `SampleList` scope